templates/docs/patterns/strip/index.md

```diff
@@ -4,70 +4,46 @@ context:
   title: Strip | Components
 ---
 
-The strip pattern provides a full width strip container in which to wrap a row. A strip can have light (`.p-strip--light`) or dark (`.p-strip--dark`) grey background.
+The strip pattern provides a full width strip container in which to wrap a grid. It is an alternative to the section component, that surrounds its content with a padding both on the top and bottom, and is used usually when a change of the background is needed or to separate different sections of the page.
 
 A `.p-strip` container should always be the parent of a `.row` (from the [Grid pattern](/docs/patterns/grid/)) and never the other way around.
 
-<div class="embedded-example"><a href="/docs/examples/patterns/strips/strips-light/" class="js-example">
-View example of the strip light pattern
-</a></div>
-
-<div class="embedded-example"><a href="/docs/examples/patterns/strips/strips-dark/" class="js-example">
-View example of the strip dark pattern
-</a></div>
+## Regular strip
 
-## White strip
+The strip component is rarely used on its own as a container with just `.p-strip` class name. It is usually combined with other variants described below to provide a specific visual style.
```

Comment:
```suggestion

Strips are containers that apply top/bottom padding and background colour. At a glance:

- Strips come in 3 sizes - shallow, regular and deep. See details below.
- They now support theming. 
- You can choose between regular (using the respective theme's background) or highlighted (Using each theme's alternative highlight colour).


The strip component is rarely used on its own as a container with just `.p-strip` class name. It is usually combined with other variants described below to provide a specific visual style.
```

Reply:
Thanks, reworded a bit to fit better in current docs structure, but I added all of the information you mentioned.